src/Main.enso

```diff
@@ -1,105 +1,193 @@
-# from Project import <NAMES>
-# from Base.Network.Http import Header # unqualified import of `Header`
-# import Base.Network.Http # qualified import of `Http`
-# from Base import all – `all` means all
+# Single line comments start with a hash.
+
+##  Documentation comments start with two hashes,and are block indented.
+    The block ends when the block indentation returns to the baseline indentation. 
+
+## Imports
+# Most Enso files will want to start with importing the standard library.
+# An unqualified import declaration, importing everything from the Base module.
 from Base import all
 
+# Unqualified imports can also be restricted
+# from Base.Network.Http import Header
 
-## the `main` method is the program entry point
-main =
-    # assignments – run the right-hand side and store the result in `operator1`
-    operator1 = 0.up_to 100 . to_vector . map .noise
-    # single-assignment! Can't modify an existing variable
-    # operator1 = 1 – reports a compile error now!
+# Qualified imports bring only the module name into
+# import Base.Network.Http 
+
+## Assignments
+# `=` for assignments, storing the result of the right-hand side under
+# the name on the left.
+x = 1
+y = 10
+result = x + y
+
+# Enso has single assignments: you cannot modify existing variable.
+# x = 2 now will result in a compile error.
+
+## Functions & Methods
+# Declare functions with a name,
+# followed by a space separated list of parameters.
+sum a b = a + b
+result2 = sum x y
```

Comment:
If you want to call sum you have to do `here.sum` now. Unless this is also supposed to be in some method body?

Reply:
Everything assigned at toplevel is a method on the current module and is called like a method (with `here.name`). Everything assigned in a method body is a variable and is accessed by name. We need to make up our minds as to which is the case in these examples.